core/io/image.cpp

```diff
@@ -3026,6 +3060,11 @@ void (*Image::_image_compress_bptc_func)(Image *, Image::UsedChannels) = nullptr
 void (*Image::_image_compress_etc1_func)(Image *) = nullptr;
 void (*Image::_image_compress_etc2_func)(Image *, Image::UsedChannels) = nullptr;
 void (*Image::_image_compress_astc_func)(Image *, Image::ASTCFormat) = nullptr;
+void (*Image::_image_compress_bc_rd_func)(Image *, Image::UsedChannels) = nullptr;
+void (*Image::_image_compress_bptc_rd_func)(Image *, Image::UsedChannels) = nullptr;
+void (*Image::_image_compress_etc1_rd_func)(Image *) = nullptr;
+void (*Image::_image_compress_etc2_rd_func)(Image *, Image::UsedChannels) = nullptr;
+//void (*Image::_image_compress_astc_rd_func)(Image *, Image::ASTCFormat) = nullptr;
```

Comment:
We tend to remove commented code. It is also unclear if betsy will have ASTC support in the short term.